hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -618,6 +624,69 @@ private void decreasePendingMetricsAndReleaseSemaphore() {
     return new XceiverClientReply(replyFuture);
   }
 
+  public XceiverClientReply sendCommandAsyncReadOnly(
```

Comment:
Since this is specific to ReadBlock for now, we can name it such. If later on we extend this to handle other commands in a similar manner with multiple response messages we can change the signature. 
```suggestion
  public XceiverClientReply sendCommandReadBlock(
```

Reply:
done